services/acm/src/main/java/software/amazon/awssdk/services/acm/auth/scheme/internal/DefaultAcmAuthSchemeProvider.java

```diff
@@ -35,6 +41,13 @@ public static DefaultAcmAuthSchemeProvider create() {
 
     @Override
     public List<AuthSchemeOption> resolveAuthScheme(AcmAuthSchemeParams authSchemeParams) {
-        return new ArrayList<>();
+        AwsV4AuthScheme awsV4AuthScheme = AwsV4AuthScheme.create();
+        return Arrays.asList(AuthSchemeOption.builder()
+                                             .schemeId(awsV4AuthScheme.schemeId())
+                                             .putSignerProperty(SERVICE_SIGNING_NAME, "acm")
+                                             .putSignerProperty(REGION_NAME, authSchemeParams.region().id())
+                                             // TODO: Make this signer property optional
+                                             .putSignerProperty(REQUEST_SIGNING_INSTANT, Instant.now())
```

Comment:
What does this signing property do? What is the point of storing an instant at client creation time?

Reply:
This value will be the requestTime(x-amz-date) used by the signer.

It won't be storing an instant at client creation time. `resolveAuthScheme` will be called for each request.

I've talked to Hayden about making this optional in the signer. However there is also the use case of accounting for TIME_OFFSET where the REQUEST_SIGNING_INSTANT value will need to be passed by caller. For that case, it can't be set here. I don't know currently where that would get set to solve for the TIME_OFFSET case - that case needs more thinking. But I'll handle that in separate PR. For now, I'll remove setting this from here, once it is made optional.